pkg/process/config/yaml_config.go

```diff
@@ -187,6 +187,11 @@ func (a *AgentConfig) LoadProcessYamlConfig(path string) error {
 		a.Windows.AddNewArgs = config.Datadog.GetBool(addArgsKey)
 	}
 
+	// Windows: Controls using the new check based on performance counters PDH APIs
+	if usePerfCountersKey := key(ns, "windows", "use_perf_counters"); config.Datadog.IsSet(usePerfCountersKey) {
```

Comment:
nit: the config has to be added as `SetKnown()` in `pkg/config/config.go` otherwise it'll report warning when agent starts and parsing this config.

Reply:
That's a good point, I forgot about it.